lib/util/trigger.php

```diff
@@ -261,16 +265,16 @@ function getAchievementPatchReadableHTML($mem, $memNotes)
 {
     $tableHeader = '
     <tr>
-      <th>ID</th>
-      <th>Flag</th>
-      <th>Type</th>
-      <th>Size</th>
-      <th>Memory</th>
-      <th>Cmp</th>
-      <th>Type</th>
-      <th>Size</th>
-      <th>Mem/Val</th>
-      <th>Hits</th>
+      <th nowrap>ID</th>
```

Comment:
The `nowrap` html attribute is outdated afaik - alternatively use `class="whitespace-nowrap"`.

Reply:
I've updated everything to use `class='whitespace-nowrap'`.

Single word cells were definitely being wrapped.
![image](https://user-images.githubusercontent.com/32680403/200190725-156c9d54-e7a5-419d-983f-7ba07116a5b1.png)

Reply:
Oh, interesting. That's odd

Reply:
Here's an extreme example when viewed from a phone:
![image](https://user-images.githubusercontent.com/32680403/200605127-98365574-36e0-4b22-8ccb-aa00130140b9.png)

No sure what it'll look like with the changes.